src/careamics/config/nm_model.py

```diff
@@ -14,43 +16,75 @@ class GaussianMixtureNMConfig(BaseModel):
     model_config = ConfigDict(
         validate_assignment=True, arbitrary_types_allowed=True, extra="allow"
     )
-    model_type: Literal["GaussianMixtureNoiseModel"]  # TODO: why do we need this?
-    """Model type."""
+    # TODO What are all these parameters?
+    # model type
+    model_type: Literal["GaussianMixtureNoiseModel"]
 
-    path: Union[Path, str, None] = None
     """Path to the directory where the trained noise model (*.npz) is saved in the
     `train` method."""
+    path: Union[Path, str, None] = None
+
+    # Path to the file containing signal or respective numpy array
+    signal: Union[str, Path, np.ndarray] = None
+
+    # Path to the file containing observation or respective numpy array
+    observation: Union[str, Path, np.ndarray] = None
 
-    weight: Any = None
-    """A [3*n_gaussian, n_coeff] sized array containing the values of the weights
-    describing the GMM noise model, with each row corresponding to one
-    parameter of each gaussian, namely [mean, standard deviation and weight].
-    Specifically, rows are organized as follows:
-    - first n_gaussian rows correspond to the means
-    - next n_gaussian rows correspond to the weights
-    - last n_gaussian rows correspond to the standard deviations
-    If `weight=None`, the weight array is initialized using the `min_signal`
-    and `max_signal` parameters."""
+    # A [3*n_gaussian, n_coeff] sized array containing the values of the weights
```

Comment:
So we use this convention (# instead of """, and above the parameter) for documenting `pydantic` attributes? I was told differently by @melisande-c. Anyway, I think we should find a common ground for this :)

Reply:
This is how it is done in the rest of the code base, maybe we can revise at some point in the future but for now let's stay consistent. This way of writing class attribute docstrings is briefly mentioned in [PEP 257](https://peps.python.org/pep-0257/): 

  _"String literals occurring elsewhere in Python code may also act as documentation. They are not recognized by the Python bytecode compiler and are not accessible as runtime object attributes (i.e. not assigned to \_\_doc\_\_), but two types of extra docstrings may be extracted by software tools:_
    
  1. _String literals occurring immediately after a simple assignment at the top level of a module, class, or \_\_init\_\_ method are called “attribute docstrings”."_

@jdeschamps, opinions?